lib/mysqlService.js

```diff
@@ -198,7 +202,7 @@ class MysqlService {
     }
 
     clean(input) {
-        return Mysql.escape(input);
+        return Mysql.escape(input, false, this._writeConnectionConfig.timezone);
```

Comment:
Config.timezone isn't necessarily defined. See mysqlPoolConnectionConfigSchema/Factory.  Feel free to get mad at me for no documentation as well :)

I'd say do a simple if/else checking existence of it.